.github/config.yml

```
@@ -24,3 +24,43 @@ requestInfoDefaultTitles:
 
 # *OPTIONAL* Label to be added to Issues and Pull Requests with insufficient information given
 requestInfoLabelToAdd: needs-more-info
+
+###
+
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thanks for opening your first issue in this project! Your input is very important to us! :hugs:
```

I considered the "us" wording and felt i divided the author from the others while I hope to communicate that we are doing this together right away if possible. I drafted a suggestion, but I'm really not sure what makes most sense.

```suggestion
  Thank you for opening your first issue in this project! Engagement like this is essential for open source projects! :hugs:
```

---

I like it!